packages/iTwinUI-react/src/types/react-table-config.ts

```diff
@@ -148,6 +148,7 @@ declare module 'react-table' {
       isScrolledToRight?: boolean;
       isScrolledToLeft?: boolean;
     };
+    lastSelectedRow?: string;
```

Comment:
Maybe rename to `lastSelectedRowId`

Reply:
**[DONE]** Renamed. Yes, I think `lastSelectedRowId` is more clearer/accurate